src/eei.cpp

```diff
@@ -744,6 +742,17 @@ inline int64_t maxCallGas(int64_t gas) {
 
       call_message.gas = gas;
 
+      if (import->base == Name("call") || import->base == Name("callCode")) {
+        try {
+          ensureSenderBalance(call_message.value);
+        } catch (OutOfGas const& e) {
```

Comment:
`e` is not used after this, shouldn't be named to avoid `unused variable` errors